CHANGES.md

```diff
@@ -5,6 +5,7 @@
 ##### Fixes :wrench:
 
 - Added a missing `CesiumRuntime.h` include in `CesiumPropertyAttribute.cpp` that broke compilation in v2.18.0 on Windows.
+- `CesiumGeoJsonDocumentRasterOverlay` now correctly uses the polygon fill style color mode instead of the line style color mode.
```

Comment:
More precise wording:

```suggestion
- Fixed a bug in `FCesiumVectorStyle` that made polygons use the color mode from `LineStyle` instead of `PolygonStyle`.
```